packages/cli/src/WorkflowHelpers.ts

```diff
@@ -566,3 +566,19 @@ export function validateWorkflowCredentialUsage(
 
 	return newWorkflowVersion;
 }
+
+//get starting node, overwise return undefined
+export function getStartNode(
+	data: IWorkflowExecutionDataProcess,
+	workflow: Workflow,
+): INode | undefined {
```

Comment:
Return types are inferrable:

```suggestion
) {
```